editor/components/color-indicator/index.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	ColorIndicator,
+} from '@wordpress/components';
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import withColorContext from '../color-palette/with-color-context';
+import { getColorName } from '../colors';
+
+export default withColorContext( ( { colors, colorValue, ariaLabel } ) => {
```

Comment:
Maybe, in this case, we don't need the  editor/components/color-indicator/index.js, the only logic operation it is doing before using components/color-indicator is the computation of the ariaLabel, components using this component can easily compute the aria-label and we avoid another component to maintain.